rest/src/test/java/com/carhouse/rest/controllerIT/CarControllerTestIT.java

```diff
@@ -38,8 +40,15 @@ void getCar() {
 
     @Test
     void getNotExistCarSale() {
```

Comment:
There's no needs to use `try-catch`. Use JUnit5 features instead.

Reply:
I've changed it